TinkeringGraphics

```diff
@@ -0,0 +1,111 @@
+#To start the prog type:   main()
```

Comment:
Use doc-strings instead of comments where appropriate. See [the PEP-257 guidelines](https://www.python.org/dev/peps/pep-0257/) for further detail.